bin/commands/install.js

```diff
@@ -1,12 +1,29 @@
-var
-  childProcess = require('child_process');
+/* eslint-disable no-console */
 
-// "kuzzle install" is an alias for "kuzzle plugins --install"
+var
+  params = require('rc')('kuzzle'),
+  Kuzzle = require('../../lib/api'),
+  RequestObject = require('kuzzle-common-objects').Models.requestObject;
 
 module.exports = function () {
-  childProcess
-    .spawn(require.main.filename, ['plugins', '--install'], {stdio: 'inherit'})
-    .on('close', code => {
-      process.exit(code);
+  var 
+    kuzzle = new Kuzzle(),
+    requestObject = new RequestObject({
+      body: {install: true}
+    });
+
+  console.log('███ kuzzle-plugins: Starting plugins installation...');
+  
+  kuzzle.start(params, {dummy: true})
```

Comment:
(or call `kuzzle.remoteActions.do('managePlugins')` instead of `kuzzle.remoteActionsController.actions.managePlugins` ?)